backend/tests/integration/records/api/test_api_mixins.py

```diff
@@ -810,3 +811,71 @@ def test_hard_delete_advisor_clears_entries_in_students_advisors_table(
     advisor.hard_delete()
 
     assert StudentsAdvisors.query.filter_by(student_id=student.id).count() == 0
+
+
+def test_journal_literature_table_is_populated_when_lit_record_added(inspire_app):
+    journal_rec = create_record("jou")
+    rec = create_record(
+        "lit", data={"publication_info": [{"journal_record": journal_rec["self"]}]}
+    )
+    assert JournalLiterature.query.filter_by(literature_uuid=rec.id).count() == 1
+
+
+def test_journal_literature_table_is_populated_when_multiple_lit_records_added(
+    inspire_app,
+):
+    journal_rec = create_record("jou")
+    journal_rec_2 = create_record("jou")
```

Comment:
I know we are using it on other places as well, but actually `create_record` is a helper function for tests and changing it might affect the tests. Hence, it would be better to use JournalRecord.create()` instead to make sure we are testing the `create` code and avoid interference with test artefacts.

Reply:
I can change it, but actually `create_record` calls the `create` method from `InspireRecord`, which is using a class method for a specified class

Reply:
The point is that indeed `create_record` just calls `InspireRecord` but it's a test logic because it's a helper function. We can leave it like that for now, since it's everywhere.

Reply:
it's ok, I've already changed it :)